src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -21,14 +23,23 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
```

Comment:
I'm not sure what that does(Maybe it is being used for logging).  I've seen *'tag'*s being used around a few files..probably all used for logging.